### PR TITLE
Add error tests

### PR DESCRIPTION
**Pull request by kshyatt**
That `convert` test is dying :( but only on `nightly` I guess

### CODE REVIEW

**Comment by andyferris** [comment]
And it's GREEN! :)

File: src/arraymath.jl [hunk 1]
```diff
@@ -111,7 +116,7 @@ end
     end
 end
 
-@inline rand!(rng::AbstractRNG, a::SA) where {SA <: StaticArray} = _rand!(rng, Size(SA), a)
+@inline rand!(rng::AbstractRNG, a::SA) where {SA <: StaticArray} = _rand!(rng, a)
```
Comment (andyferris) [inline src/arraymath.jl]: What happened to the `Size` here?

Comment (kshyatt) [inline src/arraymath.jl]: Not sure! I can put it back, or if someone wants to use the online editor...

**Comment by andyferris** [comment]
Thanks again, Katie :)

**Pull request merged**